Dockerfile

```diff
@@ -93,7 +93,8 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -qq
 RUN DEBIAN_FRONTEND=noninteractive apt-get purge lxc-docker*
 RUN DEBIAN_FRONTEND=noninteractive apt-cache policy docker-engine
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y linux-image-extra-`uname -r`
+# Includes linux-image-extra-*-generic
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends linux-generic-lts-xenial
```

Comment:
Is `--no-install-recommends` required?

Actually, checking the official install docs again ( https://docs.docker.com/engine/installation/linux/ubuntu/ ) - which changes quite frequently - it seems that `linux-image-extra-...` might no longer required, it only listed for Ubuntu 14.04 (trusty), while our non LTS stacks are on Ubuntu 16.04, and I don't see this requirement in their install docs for 14+ ubuntus.

I think upgrading the install commands to the current steps at https://docs.docker.com/engine/installation/linux/ubuntu/#install-using-the-repository should be enough, and that should work in Docker for Mac.

I'll do a quick test with this.